projects/storefrontlib/shared/components/media/media.model.ts

```diff
@@ -51,6 +66,21 @@ export interface MediaFormatSize {
   width?: number;
 }
 
+/**
+ * Specifies media queries that can be used to generate information for the
+ * browser to resolve the right media for the right layout or device.
+ */
+export type PictureElementQueries = {
```

Comment:
`medium:` let's put a link to official page with all supported queries so customers have it in one place

Reply:
Changed